service/gcs/bridge/bridge.go

```diff
@@ -45,23 +50,57 @@ func NewBridge(tport transport.Transport, coreint core.Core) *bridge {
 	}
 }
 
+// panicToError converts a panic return value into an error value.
+func panicToError(r interface{}) error {
```

Comment:
I'm new to go but when does a panic occur in go where its a good thing to recover. Panic's aren't the error model they are the exception model. It seems like panic's are likely always issues in our stack we need fixed. Am I wrong about that?